internal/receiver/smartagentreceiver/receiver.go

```diff
@@ -39,9 +52,74 @@ func NewReceiver(logger *zap.Logger, config Config, nextConsumer consumer.Metric
 }
 
 func (r *Receiver) Start(_ context.Context, host component.Host) error {
-	return nil
+	err := r.config.validate()
+	if err != nil {
+		return fmt.Errorf("config validation failed for %q: %w", r.config.Name(), err)
+	}
+
+	configCore := r.config.monitorConfig.MonitorConfigCore()
+	monitorType := configCore.Type
+	monitorName := strings.ReplaceAll(r.config.Name(), "/", "")
```

Comment:
Is / a prohibited character and that's why we replace it? Where is the monitor name visible? In the logs, in error messages? Perhaps replace by something that still looks like a delimiter but is a valid character?

Reply:
The monitor id is used as a unique internal SFx datapoint dimension value in ~collectd and elsewhere, but it may be deleted before output in all cases.  I was thinking it could in some cases be a dimension value so instinctively stripped the slash as an unsupported character in SFx.